src/components/FullDetailView/FullDetailPage.js

```diff
@@ -165,6 +171,45 @@ export class FullDetailPage extends Component {
     })
   }
 
+  handleRevert(value) {
+    const { uiCurateResetDefinitionPreview } = this.props
+    const { changes } = this.state
+    if (isEmpty(changes)) return
+    if (value) {
+      const revertedChanges = omitBy(changes, (_, index) => index.startsWith(value))
+      this.setState({ changes: revertedChanges, sequence: this.state.sequence + 1 }, () => this.previewDefinition())
+      return
+    }
+    const key = `open${Date.now()}`
+    const NotificationButtons = (
+      <Fragment>
+        <AntdButton
+          type="primary"
+          size="small"
+          onClick={() =>
+            this.setState({ changes: {} }, () => {
+              uiCurateResetDefinitionPreview()
+              notification.close(key)
+            })
+          }
+        >
+          Confirm
+        </AntdButton>
+        <AntdButton type="secondary" size="small" onClick={() => notification.close(key)}>
+          Dismiss Notification
+        </AntdButton>
+      </Fragment>
+    )
+    notification.open({
```

Comment:
still a bit fussed about this pattern of `open()` rather than having a prop like `show: true`. Same for `close`

Reply:
The [Notification component](https://ant.design/components/notification/) in antd-design is actually an API. It requires to call those methods (open and close) in order to show or hide the notification